testsupport/wait/member.go

```diff
@@ -380,6 +383,13 @@ func (a *MemberAwaitility) WaitForClusterResourceQuota(name string) (*quotav1.Cl
 		}
 		a.T.Logf("found ClusterResourceQuota '%s'", name)
 		quota = obj
+
+		for _, match := range criteria {
+			if !match(a, *obj) {
+				return false, nil
+			}
+		}
```

Comment:
It's not critical, but I would add the check before it prints that the CRQ was found
```suggestion
		for _, match := range criteria {
			if !match(a, *obj) {
				a.T.Logf("Waiting until ClusterResourceQuota '%s' meets the given criteria", name)
				return false, nil
			}
		}
		a.T.Logf("found ClusterResourceQuota '%s'", name)
		quota = obj
```